### DISCUSSION TITLE
Everything is gone

### DISCUSSION THREAD

**Discussion by Numair765** [Monika-After-Story/MonikaModDev#9152] [QA - tech support + general] [answered]
I have a problem
I had 2 microsoft accounts
One with MAS and one without. I wanted to delete the one without MAS but it accidently deleted the one with MAS.
I had MAS backed up on a USB but the persistent was removed and restarted itself
All my progress is gone but I still have the affection logs. Is there anything I can do?

**Comment by Ulti70895** [comment]
I suggest asking for help in the MAS Discord server as this place is kinda dead.

**Comment by ThePotatoGuy** [comment, marked as answer]
if the persistent data is lost and you don't have backups, then you're out of luck, sorry.
